src/WebUI/Pods/PodsTable.tsx

```diff
@@ -120,7 +120,13 @@ export class PodsTable extends BaseComponent<IPodsTableProperties> {
             this.props.onItemActivated(event, selectedItem);
         }
         else if (selectedItem) {
-            ActionsHubManager.GetActionsHub<SelectionActions>(SelectionActions).selectItem.invoke({ item: selectedItem, showSelectedItem: true, selectedItemType: SelectedItemKeys.OrphanPodKey });
+            ActionsHubManager.GetActionsHub<SelectionActions>(SelectionActions).selectItem.invoke(
+                { 
+                    item: selectedItem, 
+                    itemName: selectedItem.metadata.name,
```

Comment:
should we check for selectedItem.metadata nullability?

Reply:
"Every object created over the whole lifetime of a Kubernetes cluster has a distinct UID".
Changed this to use metadata.UID, so it might not be needed as UID is guaranteed to be present.